CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 1.13
+* Tests now ensure that composer.lock is kept up to date. (GitHub #132)
+* The page title block is now properly placed in the Seven theme. (GitHub #190)
+* Twitter media entities now have automatically generated thumbnails.
```

Comment:
Not quite accurate -- we should say that it can be configured to generate thumbnails for textual tweets if you configure it to use the Twitter API, or something like that.